spectree/plugins/falcon_plugin.py

```diff
@@ -182,12 +182,32 @@ def request_validation(self, req, query, json, headers, cookies):
             media = None
         if json:
             req.context.json = json.parse_obj(media)
+        elif form_data and media is not None:
+            data = {}
+            for part in media:  # TODO: add support for falcon 2.0 (media is always None for 'multipart/form-data')
+                # Compare first part of content_type with media types
+                if part.content_type.split('/')[0] == 'text':
+                    data[part.name] = part.text
+                elif part.content_type.split('/')[0] in ['image', 'application', 'audio']:
+                    data[part.name] = {
+                        "filename": part.filename,
+                        "name": part.name,
+                        # "content_length": len(part.data),  # FIXME: raises error if file is too big
+                        # sadly, BufferedReader don't have 'tell' method
+                        "content_length": 0,  # TODO: replace placeholder with something that will work
+                        "content_type": part.content_type,
+                        "stream": part.stream.read(),
+                    }
+                # TODO: add support for other media types?
+                # TODO: OR handle unsupported media types?
```

Comment:
Refs: https://www.iana.org/assignments/media-types/media-types.xhtml

How about the following?

* font
* video
* message
* model

I wonder what's the major difference among these types for file handlers?